ocs_ci/deployment/acm.py

```
@@ -188,8 +194,138 @@ def deploy_downstream(self):
+        For every non-ACM cluster:
+          1. Switch to the hub context and patch the SubmarinerConfig
+             (namespace = cluster_name, resource_name = "submariner") with the
+             routeagent image override.
+          2. Switch to the managed cluster context and wait for the
+             routeagent DaemonSet rollout to complete.
+
+        TODO: Remove after submariner 0.24.1
+        """
+        routeagent_image = "quay.io/yboaron/submariner-route-agent:dev-20260811"
+        logger.info(
+            f"[submariner#4124 WA] Overriding routeagent image to {routeagent_image}"
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)